tests/test_secure_boot.py

```diff
@@ -25,4 +25,15 @@ def test_secure_boot_enabled(self, connection, conversion):
             pytest.skip("MEN-5253: Secure Boot not yet working for Yocto")
 
         output = connection.run("mokutil --sb-state").stdout.strip()
+
+        if conversion:
+            grub_d_integration = connection.run(
+                "test -e /etc/grub.d/00_05_mender_setup_env_grub && echo on || echo off"
+            )
```

Comment:
This will work.

Have you considered passing it as a test variable (adding it [here](https://github.com/mendersoftware/mender-convert/blob/93953ad9e5478f1261163b9f9c75a8ed2ab346fe/mender-convert-package#L400) and then parsing it in the fixture somehow)? Is it too much work? Relying on the file present is an indirect way (workaround?) to figure out if Secure Boot is enabled and generally it will be better though and explicit variable or a feature flag.

Reply:
You're right, that's cleaner. I'll change it (or at least try).